deploy/crds/hwameistor.io_localstoragenodes_crd.yaml

```diff
@@ -100,6 +100,107 @@ spec:
+                            description: Capacity of the disk in bytes
+                            format: int64
+                            type: integer
+                          diskType:
+                            description: DiskType represents the type of drive like
+                              SSD, HDD etc.,
+                            type: string
+                        type: object
+                      diskRefs:
+                        description: DiskRefs represents which disks are assigned
+                          to the LocalDiskClaim
+                        items:
+                          description: 'ObjectReference contains enough information
+                            to let you inspect or modify the referred object. ---
+                            New uses of this type are discouraged because of difficulty
+                            describing its usage when embedded in APIs.  1. Ignored
+                            fields.  It includes many fields which are not generally
+                            honored.  For instance, ResourceVersion and FieldPath
+                            are both very rarely valid in actual usage.  2. Invalid
+                            usage help.  It is impossible to add specific help for
+                            individual usage.  In most embedded usages, there are
+                            particular     restrictions like, "must refer only to
+                            types A and B" or "UID not honored" or "name must be restricted".     Those
+                            cannot be well described when embedded.  3. Inconsistent
+                            validation.  Because the usages are different, the validation
+                            rules are different by usage, which makes it hard for
+                            users to predict what will happen.  4. The fields are
+                            both imprecise and overly precise.  Kind is not a precise
+                            mapping to a URL. This can produce ambiguity     during
+                            interpretation and require a REST mapping.  In most cases,
+                            the dependency is on the group,resource tuple     and
+                            the version of the actual struct is irrelevant.  5. We
+                            cannot easily change it.  Because this type is embedded
+                            in many locations, updates to this type     will affect
+                            numerous schemas.  Don''t make new APIs embed an underspecified
+                            API type they do not control. Instead of using this type,
+                            create a locally provided and used type that is well-focused
+                            on your reference. For example, ServiceReferences for
+                            admission registration: https://github.com/kubernetes/api/blob/release-1.17/admissionregistration/v1/types.go#L533
```

Comment:
```suggestion
                          description: 'ObjectReference contains sufficient information
                            to let you inspect or modify the referred object. ---
                            New uses of this type are discouraged because of difficulty
                            describing its usage when embedded in APIs. 1. Ignored
                            fields.  It includes many fields which are not generally
                            honored.  For instance, ResourceVersion and FieldPath
                            are both very rarely valid in actual usage.  2. Invalid
                            usage help.  It is impossible to add specific help for
                            individual usage.  In most embedded usages, there are
                            particular restrictions like, "must refer only to types A
                            and B" or "UID not honored" or "name must be restricted". 
                            Those cannot be well described when embedded. 
                            3. Inconsistent validation.  Because the usages vary,
                            the validation rules are different by usage, which makes it
                            hard for users to predict what will happen.  4. The fields are
                            both imprecise and overly precise.  Kind is not a precise
                            mapping to a URL. This can produce ambiguity during
                            interpretation and require a REST mapping.  In most cases,
                            the dependency is on the group, resource tuple and
                            the version of the actual struct is irrelevant.  5. We
                            cannot easily change it.  Because this type is embedded
                            in many locations, updates to this type will affect many
                            schemas.  Do not make new APIs embed with an underspecified
                            API type that is out of control. Instead of using this type,
                            create a locally provided and used type that is well-focused
                            on your reference. For example, ServiceReferences for
                            admission registration: https://github.com/kubernetes/api/blob/release-1.17/admissionregistration/v1/types.go#L533
```